.auto-changelog

```diff
@@ -0,0 +1,4 @@
+{
+    "output": "CHANGELOG.md",
+    "template": "keepachangelog"
+}
```

Comment:
Although JS does not _require_ trailing newlines, currently we have configured in `.editorconfig` that all files should have final newline.

This might be changed, but if changed, it should be done globally.

Reply:
We already had a discussion about that: https://github.com/romcal/romcal/pull/6#discussion_r256801590
As I already said, that one is very subjective. And I don't see the advantage of changing this option. It's not about something required or not but a convenient option when you write code.

From my point of view I prefer to have this new line. But what is most important is to respect this setup. I often go from one project to another that have different configuration options. And thanks to the `.editorconfig` file, my IDE adapts itself automatically.

Reply:
Yes, we had that discussion, but now don’t suggest to _remove_ the final newline, but _insert_. :smiley: Sorry that wasn’t clear enough.

And I agree on `.editorconfig` usage; I use it as well. 👍

Reply:
Sorry, I read to fast 😉

Reply:
I've installed the `editorconfig` extension in my IDE to take advantage of the rules defined in `.editorconfig`. I agree with the setup and will endeavour to follow it in the future.